### PR TITLE
Docs: Local dev refactoring

### PR DESCRIPTION
**Pull request by kiwicopple**

### CODE REVIEW

File: apps/docs/pages/guides/cli/seeding-your-database.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Using the seed file
+
+```
+supabase/seed.sql
+```
+
+## Generating seed data from Production
+
+- Use snaplet
```
Comment (peterp) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:22]:
```suggestion
## Generating seed data 

Snaplet's can generate seed files based off of a database schema and a broad-stroke definition of what data should be generated, as an example:

`// TODO: Example of generate definition`

Run `snaplet config generate --sql > seed.sql` to generate a seed script.

`// TODO: Example of generated output`

[Learn more in Snaplet's Docs](https://docs.snaplet.dev/references/data-operations/generate)

```

File: apps/docs/pages/guides/cli/seeding-your-database.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  ('Canada', 'CA'),
+  ('Mexico', 'MX');
+```
+
+### Splitting up your seed file
+
+The seed file can grow quite large, so you can split it up into multiple files. For example, you can create a `seed` folder and add a `countries.sql` file. Then you can add the following to your `seed.sql` file.
+
+```sql supabase/seed.sql
+\i seed/countries.sql
```
Comment (kiwicopple) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:36]: Error: 

```
ERROR: syntax error at or near "\" (SQLSTATE 42601)
```

File: apps/docs/pages/guides/cli/seeding-your-database.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+```
+
+### Using CSV data to seed your database
+
+co
+You can use the `COPY` command to import CSV data into your database. For example, you can create a `seed` folder and add a `countries.csv` file. Then you can add the following to your `seed.sql` file.
+
+```sql supabase/seed.sql
+COPY countries (name, code) FROM 'seed/countries.csv' WITH (FORMAT csv);
```
Comment (kiwicopple) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:45]: error: 

```
ERROR: must be superuser or have privileges of the pg_read_server_files role to COPY from a file (SQLSTATE 42501)
```

File: apps/docs/pages/guides/cli.mdx [hunk 4]
```diff
@@ -1,162 +1,47 @@
 import Layout from '~/layouts/DefaultGuideLayout'
 
 export const meta = {
-  title: 'Supabase CLI',
-  description:
-    'The Supabase CLI provides tools to develop your project locally and deploy to the Supabase Platform.',
-  subtitle:
-    'The Supabase CLI provides tools to develop your project locally and deploy to the Supabase Platform.',
+  id: 'cli',
+  title: 'Local Dev / CLI',
```
Comment (sweatybridge) [inline apps/docs/pages/guides/cli.mdx:5]: Nitpick: the forward slash feels a bit out of place. How about 

```suggestion
  title: 'Local Dev with CLI',
```

Or perhaps just Local Dev?

**Review by sweatybridge — APPROVED** [review]

**Pull request merged**